pkg/command/discovery_source_test.go

```diff
@@ -81,6 +84,47 @@ func Test_createDiscoverySource(t *testing.T) {
 	assert.Contains(err.Error(), "unknown discovery source type 'unexpectedValue'")
 }
 
+// Test_createAndListDiscoverySources test 'tanzu plugin source list' when TANZU_CLI_ADDITIONAL_PLUGIN_DISCOVERY_IMAGES_TEST_ONLY has set test only discovery sources
+func Test_createAndListDiscoverySources(t *testing.T) {
+
+	// Initialize the plugin source
+	rootCmd, err := NewRootCmd()
+	assert.Nil(t, err)
+	rootCmd.SetArgs([]string{"plugin", "source", "init"})
+	err = rootCmd.Execute()
+	assert.Nil(t, err)
+
+	// List plugin source - one
+	os.Setenv("TANZU_CLI_ADDITIONAL_PLUGIN_DISCOVERY_IMAGES_TEST_ONLY", "harbor-repo.vmware.com/tanzu_cli_stage/plugins/plugin-inventory:latest")
```

Comment:
Can you use `constants.ConfigVariableAdditionalDiscoveryForTesting` instead of `"constants.ConfigVariableAdditionalDiscoveryForTesting"`, here and below?